### DISCUSSION TITLE
Changing port of cottontail

### DISCUSSION THREAD

**Discussion by RibinMathewChalumattu** [vitrivr/cottontaildb#63] [Q&A] [answered]
Dear devs,

Where can I change the port cottontail runs on? I haven't seen a port option in the provided _config.json_ file.

Best,

Ribin

**Comment by ppanopticon** [comment, marked as answer]
Hello Ribin

Just add, _server.port_ property to the config.json

```json
{
  "root": "./cottontaildb-data",
  "server": {
     "port": 1234
  },
  "mapdb": {
    "enableMmap": true,
    "forceUnmap": true,
    "pageShift": 22
  }
}
```
Hope this helps.

Best,
Ralph
